src/core/contracts/identity.test.ts

```diff
@@ -331,74 +339,358 @@ describe("identity", () => {
     });
   });
 
-  describe("#getRemoveDelegateLogData", () => {
-    let contractAddress: EthereumAddress;
-    let contractOwner: EthereumAddress;
-    let blockNumberForRemoval: number;
+  describe("#getDelegeteIdentitiesFor", () => {
```

Comment:
Delegate has a typo here

Reply:
dang. Iol. I thought I misspelled it the first time.